### ISSUE TITLE
`generic.ci.security.use-frozen-lockfile.use-frozen-lockfile-npm` causes false positive

### ISSUE THREAD

**Issue by ianmuscat** [returntocorp/sgrep-rules#2622]
**Describe the bug**
`generic.ci.security.use-frozen-lockfile.use-frozen-lockfile-npm` causes false positives when [pnpm](https://pnpm.io/) is used.

**To Reproduce**
```
pnpm install
```

**Expected behavior**
The rule should not trigger since pnpm can detect when it is [running in a CI environment](https://pnpm.io/cli/install#--frozen-lockfile).

**Priority**
How important is this to you?
- _*P2*: annoying but not blocking me_

**Comment by 0xDC0DE** [comment]
Thanks for your report, @ianmuscat , should be fixed pretty soon!

**Comment by LewisArdern** [comment]
We merged a fix for this
